pkg/reconcilermanager/controllers/reposync_controller.go

```diff
@@ -934,16 +934,24 @@ func (r *RepoSyncReconciler) validateNamespaceSecret(ctx context.Context, repoSy
 	return validateSecretData(authType, secret)
 }
 
-func (r *RepoSyncReconciler) upsertRoleBinding(ctx context.Context, reconcilerRef, rsRef types.NamespacedName) (client.ObjectKey, error) {
+func (r *RepoSyncReconciler) upsertRoleBinding(ctx context.Context, reconcilerRef, rsRef types.NamespacedName, labelMap map[string]string) (client.ObjectKey, error) {
 	rbRef := client.ObjectKey{
 		Namespace: rsRef.Namespace,
 		Name:      RepoSyncPermissionsName(),
 	}
 	childRB := &rbacv1.RoleBinding{}
 	childRB.Name = rbRef.Name
 	childRB.Namespace = rbRef.Namespace
+	// copy labelMap
```

Comment:
nit: if you rebase you can use `ManagedObjectLabelMap(r.syncKind, rsRef)` to create a new map with the standard labels, which may simplify this snippet

This could also use a comment explaining why we do not include the `sync-name` label here. For example:
```
// Omit the sync-name label, because the RoleBinding may be shared by multiple RepoSyncs
```